src/NUnitConsole/nunit3-console.tests/ResultReporterTests.cs

```diff
@@ -96,12 +91,26 @@ public void ReportSequenceTest()
         }
 
         [Test]
+        public void CheckReportSettingsTest()
+        {
+            var expected = new [] {
+                "Run Settings",
+                "    Setting1Name: Setting1Value",
+                "    Setting2Name: Setting2Value",
+                ""
+            };
+
+            var actualSettingsReport  = GetReportLines(_reporter.WriteRunSettingsReport);
+            Assert.That(expected, Is.EqualTo(actualSettingsReport));
+        }
+
+        [Test]
         public void SummaryReportTest()
         {
-            var expected = new string[] {
+            var expected = new [] {
```

Comment:
I use implicit typed arrays throughout  the CF code, but that, by definition, requires C# 3.0 (VS2008).

So the question is, are we going to support any C# versions < 3.0?  Software like mono and cecil condition for earlier frameworks and compilers.  We could do that, but it would be messy.  It would seem to be unnecessary in this case, since it is easy to just use explicit typed arrays.  There is no difference in the generated code.